ember/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM danlynn/ember-cli:latest
+COPY . /myapp
+RUN yarn build
```

Comment:
I think this also needs a `yarn install` no?

Reply:
Done.